packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -303,16 +310,14 @@ async function fetchLatestSnapshotCore(
                     sequenceNumber,
                     ops: snapshot.ops?.length ?? 0,
                     headers: Object.keys(response.requestHeaders).length !== 0 ? true : undefined,
-                    redirecttime: redirectTime,
-                    dnsLookuptime: dnstime,
-                    responsenetworkTime: responseTime,
-                    tcphandshakeTime: tcpHandshakeTime,
-                    secureconnectiontime: secureConntime,
-                    fetchstarttorespendtime: fetchStToRespEndTime,
-                    reqstarttorespendtime: reqStToRespEndTime,
-                    overalltime: overallTime,
-                    networktime: networkTime,
-                    clienttime: clientTime,
+                    redirectTime: redirectTime,
```

Comment:
I think compiler should fail and ask to use more concise form:
```ts
redirectTime,
```
It might be worth taking a look at other variable names and rename them to be what property in telemetry should look like to make code more compact.

Reply:
renamed them.